SOURCES/99-enable-sm-driver-linstor.conf

```diff
@@ -1,2 +1,2 @@
 # DO NOT MODIFY: Add sm linstor plugin.
-sm-plugins=ext nfs iscsi lvmoiscsi dummy file hba rawhba udev iso lvm lvmohba lvmofcoe shm smb cephfs glusterfs xfs zfs ext4 linstor
+sm-plugins=ext nfs iscsi lvmoiscsi dummy file hba rawhba udev iso lvm lvmohba lvmofcoe shm smb cephfs glusterfs xfs zfs ext4 largeblock linstor 
```

Comment:
Sorry for the extra nitpicking, but there's an extraneous space at the end of the line.

Reply:
Oops. Sorry, it's fixed now.